dlp/src/main/java/dlp/snippets/InspectImageFile.java

```diff
@@ -33,14 +33,6 @@
 import java.util.List;
 
 public class InspectImageFile {
-
-  public static void inspectString() {
-    // TODO(developer): Replace these variables before running the sample.
-    String projectId = "my-project-id";
-    String textToInspect = "My name is Gary and my email is gary@example.com";
-    inspectString(projectId, textToInspect);
```

Comment:
@kurtisvg this was breaking the build as it could not compile. This inspectString method does not exist in this class.

Reply:
Can we update to the following instead?

```java
public static void inspectImageFile() {	
    // TODO(developer): Replace these variables before running the sample.	
    String projectId = "my-project-id";	
    String filePath = "path/to/image/file";	
    inspectString(projectId, filePath);
 }
```

Reply:
Sorry, took another look and now I think I know what this sample is doing. I believe the correct fix is:
```
public static void inspectImageFile() {	
    // TODO(developer): Replace these variables before running the sample.	
    String projectId = "my-project-id";	
    String filePath = "path/to/image/file";	
    inspectImageFile(projectId, filePath);
 }
```

where `inspectImageFile(String, String)` is found directly below this in this class. Let me know if my understanding is incorrect.

Reply:
That looks correct to me 👍